### PR TITLE
Catch exception when tube not found

### PR DESCRIPTION
**Pull request by lanffy**
Catch exception when tube not found for ``beanstalkd/api/tubes/{tube}`` and ``beanstalkd/tubes/{tube}``.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /vendor/
 node_modules/
+/.idea
```
Comment (ksassnowski) [inline .gitignore]: IDE specific folders should be in your system's global `.gitignore` not in the project `.gitignore`. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.1] - 2018-08-17
+### Added
+- Catch exception when tube not found for ``beanstalkd/api/tubes/{tube}`` and ``beanstalkd/tubes/{tube}``.
```
Comment (ksassnowski) [inline CHANGELOG.md]: Please don't update the changelog in this changeset. I will update it before releasing a new version. The release date as well as the added/changed features might still change.

File: composer.json [hunk 3]
```diff
@@ -1,16 +1,15 @@
 {
-    "name": "dionera/laravel-beanstalkd-ui",
+    "name": "lanffy/laravel-beanstalkd-ui",
```
Comment (ksassnowski) [inline composer.json]: Please don't change the package name.

File: composer.json [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
     "description": "An Admin UI for Beanstalkd and Laravel",
     "license": "MIT",
     "authors": [
         {
-            "name": "Kai Sassnowski",
-            "email": "ksassnowski@gmail.com"
+            "name": "Lanffy",
+            "email": "rlanffy@gmail.com"
```
Comment (ksassnowski) [inline composer.json]: Please don't change the author. You will be listed as a contributor in the repository instead.

File: composer.json [hunk 5]
```diff
@@ -1,16 +1,15 @@
...
         }
     ],
     "require": {
         "php": "^7.0",
-        "laravel/framework": "~5.5",
```
Comment (ksassnowski) [inline composer.json]: Any particular reason this dependency got removed?

File: src/Controllers/Api/StatsController.php [hunk 6]
```diff
@@ -36,7 +36,11 @@ public function __construct(PheanstalkInterface $pheanstalk, JobRepository $jobs
      */
     public function statsForTube($tube)
     {
-        $tubeStats = $this->pheanstalk->statsTube($tube);
+        try {
+            $tubeStats = $this->pheanstalk->statsTube($tube);
+        } catch (\Exception $e) {
+            return json_encode(['message' => $e->getMessage(), 'tube name:' => $tube]);
```
Comment (ksassnowski) [inline src/Controllers/Api/StatsController.php]: We should change this to use Laravel's response methods. Something like

```php
$data = json_encode([
    'message' => $e->getMessage(), 
    'tube name:' => $tube
]);

return response()->json($data, 200);
```

File: src/Controllers/TubesController.php [hunk 7]
```diff
@@ -53,7 +53,11 @@ public function index()
      */
     public function showTube($tube)
     {
-        $stats = $this->pheanstalk->statsTube($tube);
+        try {
+            $stats = $this->pheanstalk->statsTube($tube);
+        } catch (\Exception $e) {
+            return json_encode(['message' => $e->getMessage(), 'tube name:' => $tube]);
```
Comment (ksassnowski) [inline src/Controllers/TubesController.php]: See above. Use `response()->json()`

**Comment by lanffy** [comment]
You let me know more about this,Thank you very much.

**Pull request closed without merging**